examples/building_and_installing_viame/README.rst

```diff
@@ -214,8 +207,9 @@ Building on Windows
 
 Building on windows can be very similar to Linux if using a shell like cygwin
 (``https://www.cygwin.com/``), though if not you may want to go grab the GUI
-ersions of CMake (``https://cmake.org/``) and TortoiseGit (``https://tortoisegit.org/``).
-Currently Visual Studio 2015 and some versions of 2017 are supported.
+versions of CMake (``https://cmake.org/``) and TortoiseGit (``https://tortoisegit.org/``).
+Currently Visual Studio 2013, 2015 and EARLY VERSION ONLY of 2017 are supported.
```

Comment:
I thought we dropped support for 2013?
Can you check if the latest 2017 actually works?

Reply:
That's a bad merge from an earlier version. Thought the conflict was just from that missing v I fixed, didn't see the version list had changed, too. I'll switch that line back.